pkg/operator/ceph/cluster/osd/create.go

```diff
@@ -94,6 +95,12 @@ func (c *createConfig) createNewOSDsFromStatus(
 			logger.Debugf("not creating deployment for OSD %d which already exists", osd.ID)
 			continue
 		}
+
+		// osd prepare jobs don't generate cephx status info for OSDs. since this is a new OSD
+		// deployment, generate a first-deploy cephx status
+		status.OSDs[i].CephxStatus = keyring.UpdatedCephxStatus(false, c.cluster.spec.Security.CephX.Daemon,
```

Comment:
This status for individual OSDs is only in memory and not stored on the cluster CR status, right?